### DISCUSSION TITLE
Install on  the pilot and Rasbian Light

### DISCUSSION THREAD

**Discussion by neurodavidus** [wehr-lab/autopilot#76] [Q&A] [answered]
Hello there, 
Some more  basic&general questions during my baby steps in getting starting with autopilot.
I founded it complicated to reinstall Raspbian Lite on my brand new Raspberry PI (I did not have handy an empty SD card or a sd card reader on my labtop). I guess it is ok to install autopilot on the pilot with the OS already installed on the Raspberry PI, correct ? Or would that really affect the performance of the pilot later ?
Thanks for your advice!
DAvid

**Comment by sneakers-the-rat** [comment, marked as answer]
Baby steps are the best steps! 

Yes, it's possible to run autopilot on regular raspberry pi OS, there are some minor global performance deficits just because the pi has more background processes running, but it should still run :). 

You can get to a lite-ish OS from regular OS by disabling booting to desktop using `sudo raspi-config` and disabling unused system services, you can list systemd services with `sudo systemctl list-units --type=service` and then stopping & disabling any of them with `sudo systemctl stop service-name && sudo systemctl disable service-name`
